plugins/home/README.md

```diff
@@ -66,23 +66,171 @@ In summary: it is not necessary to use the `createCardExtension` extension creat
 Composing a Home Page is no different from creating a regular React Component, i.e. the App Integrator is free to include whatever content they like. However, there are components developed with the Home Page in mind, as described in the previous section. If created by the `createCardExtension` extension creator, they are rendered like so
 
 ```tsx
-import React from 'react'
-import Grid from '@material-ui/core/Grid'
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
 import { RandomJokeHomePageComponent } from '@backstage/plugin-home';
 
 export const HomePage = () => {
   return (
     <Grid container spacing={3}>
       <Grid item xs={12} md={4}>
-        <RandomJokeHomePageComponent>
+        <RandomJokeHomePageComponent />
       </Grid>
     </Grid>
-  )
-}
+  );
+};
 ```
 
 Additionally, the App Integrator is provided an escape hatch in case the way the card is rendered does not fit their requirements. They may optionally pass the `Renderer`-prop, which will receive the `title`, `content` and optionally `actions`, `settings` and `contextProvider`, if they exist for the component. This allows the App Integrator to render the content in any way they want.
 
+## Customizable home page
+
+If you want to allow users to customize the components that are shown in the home page, you can use CustomHomePageGrid component.
+By adding the allowed components inside the grid, the user can add, configure, remove and move the components around in their
+home page. The user configuration is also saved and restored in the process for later use.
+
+```tsx
+import {
+  HomePageRandomJoke,
+  HomePageStarredEntities,
+  CustomHomepageGrid,
+} from '@backstage/plugin-home';
+import { Content, Header, Page } from '@backstage/core-components';
+import { HomePageSearchBar } from '@backstage/plugin-search';
+import { HomePageCalendar } from '@backstage/plugin-gcalendar';
+import { MicrosoftCalendarCard } from '@backstage/plugin-microsoft-calendar';
+
+export const HomePage = () => {
+  return (
+    <CustomHomepageGrid>
+      // Insert the allowed widgets inside the grid
+      <HomePageSearchBar />
+      <HomePageRandomJoke />
+      <HomePageCalendar />
+      <MicrosoftCalendarCard />
+      <HomePageStarredEntities />
+    </CustomHomepageGrid>
+  );
+};
+```
+
+### Creating Customizable Components
```

Comment:
Oh that's a good point, I'm also happy to defer to a separate PR!

Reply:
Thanks! Yeah, probably they should be in the storybook as well (though I have no idea how that works... :D). Now it's time to iterate the functionality in case it needs something and also provide more widgets that people can add to the home page. I have already a couple in my mind :)